container_test.go

```diff
@@ -72,7 +72,7 @@ func TestRegister(t *testing.T) {
 			err := c.Provide(tc.param)
 
 			if tc.err != nil {
-				require.EqualError(t, err, tc.err.Error())
+				require.Contains(t, err.Error(), tc.err.Error())
```

Comment:
Can we add a test that check, that function name is present in the error message?